src/components/Tooltip.tsx

```diff
@@ -71,14 +71,15 @@ function Tooltip({
         event.currentTarget.addEventListener('keyup', handleKeyUp, { once: true })
         // Prevent tooltip.show from being called by TooltipReference
         event.preventDefault()
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
         if (typeof child.props.onFocus === 'function') child.props.onFocus(event)
```

Comment:
I'm not sure I understand why this call is unsafe.

Reply:
I tried answering your question and I couldn't without digging in :| I'm afraid the reasons are varied and not obvious, but  the top issue is that `child.props` is `any`, and `any` could be `null` or `undefined`. The check would be valid if child.props were something like `Record<string, unknown>` but I don't know if it's even doable.

Reply:
And if we got `any` out of the way, the answer probably boils down to https://github.com/microsoft/TypeScript/issues/21732